pages/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from 'react'
+import { Head, Button, HorizontalLayout } from '../components/Core'
+import {
+  getMaintainer,
+  getSkills,
+  getAchievements,
+  getMinAchievementDate,
+  getHighlights,
+  getContactInformation } from '../lib/data'
+import {
+  Header,
+  Intro,
+  Skills,
+  Achievements,
+  Highlights, 
+  ContactInformation } from '../components/Home'
+import styles from '../styles/Home.module.css'
+
+const Home: FC<DataProps> = ({
+  maintainer,
+  skills,
+  achievements,
+  highlights,
+  contactInformation
+  }) => {
+
+  const renderContent = () => {
+      return (
+        <>
+          <Header name={maintainer.name} headline={maintainer.headline} image={maintainer.image} />
+          <br />
```

Comment:
using <br> is not a good pattern. Your elements themselves should be contained in a manner they respect box model and margin collapse and normal flow in the page